### PR TITLE
Completed api-authorization.

### PR DESCRIPTION
**Pull request by stefancarrera**
api-author

### CODE REVIEW

File: api-authorization/server/authorization-middleware.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  try {
+    const token = req.headers['x-access-token'];
+    if (!token) {
+      throw new ClientError(401, 'authentication required');
+    }
+    const payload = jwt.verify(token, process.env.TOKEN_SECRET);
+    req.user = payload;
+    next();
+  } catch (err) {
+
```
Comment (scbowler) [inline api-authorization/server/authorization-middleware.js:15]: If you use a `try` `catch` you must call `next` within the `catch` and pass it the `err`. If you don't your server will hang if there is an error and never send anything back to the client.

```JAVASCRIPT
catch (err) {
  next(err);
}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one update needed. Let me know if you have any questions

**Comment by stefancarrera** [comment]
Added next(err);

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**